src/foam/comics/v2/DAOSummaryView.js

```diff
@@ -217,7 +217,7 @@ foam.CLASS({
         if ( ! this.stack ) return;
         let newRecord = this.data.clone();
         // Clear PK so DAO can generate a new unique one
-        newRecord.id = undefined;
+        newRecord.instance_.id = undefined;
```

Comment:
Why are you doing this?

Reply:
@kgrgreer we were debugging it the other day, and for some reason `newRecord.id = undefined` wasn't clearing the id.  I'm assuming this fixed that issue.

Reply:
And if you test it now, is this still required? And if so, what class is 'newRecord'?

Reply:
Yes, it is required. (if we have Id, it will be updated)
'newRecord' is an FObject. It is a new copy of the FObject that will be send it to the server.